### PR TITLE
Chore/upgrade packages

### PR DESCRIPTION
**Pull request by wanbinkimoon**
# upgrade packages

upgraded as follows
Screenshot 2021-02-26 at 17 51 01

----

### Motivation and Context

Maintenance 

### Priority

- [ ] 1 - Highest
- [x] 2 - High
- [ ] 3 - Medium
- [ ] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Component (add a Component as approved dy Design System)
- [ ] Docs (add documentation)
- [x] Chore (changes that adds small enhancement)
- [ ] Breaking (fix or feature that would cause existing functionality to not work as expected)

### Abstract

N/A

### Screenshots
N/A

### Note

----

## Component and Fix approval flow to Master branch

A Pull Request to be merged must have two approvals, one from a member of the product team who developed it and another from a member of the dst dev team other than the team representative.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "name": "@zanichelli/albe-web-components",
   "version": "1.4.1",
   "description": "The Web Components implementation of Albe, the Zanichelli's design system.",
-  "main": "dist/index.js",
-  "module": "dist/index.mjs",
+  "main": "dist/index.cjs.js",
```
Comment (ufollettu) [inline package.json:5]: why did you need to change these values?

**Review by ufollettu — COMMENTED** [review]
running `yarn test` I found a bunch of console.warn and a TypeError. Please check
running `yarn test.e2e` there's some console.warn too Please check

**Comment by wanbinkimoon** [comment]
As discussed in async there are stilla couple of warning which the cause is not entirely clear to me

**Comment by wanbinkimoon** [comment]
`yarn build` 👍 
`yarn test` 👍 
`yarn test.e2e` 👍 
`yarn list typescript` => `typescript@4.2.3`  👍 

LGTM
